[testnet][fullnode] panic: interface conversion: interface {} is []interface {}, not string

```
panic: interface conversion: interface {} is []interface {}, not string
goroutine 15186371 [running]:
github.com/incognitochain/incognito-chain/rpcserver.(*WsServer).handleSubcribeCrossCustomTokenPrivacyByPrivateKey(0xc000120700, 0x1186fe0, 0xc00911f1a0, 0xc1c91fa738, 0x2, 0xc204598a20, 0xc2045989c0)
	/root/go/src/github.com/incognitochain/incognito-chain/rpcserver/ws_account.go:197 +0x1061
created by github.com/incognitochain/incognito-chain/rpcserver.(*WsServer).subscribe
	/root/go/src/github.com/incognitochain/incognito-chain/rpcserver/websocket.go:227 +0x346
```

--- a/rpcserver/ws_account.go
+++ b/rpcserver/ws_account.go
@@ -194,7 +194,12 @@ func (wsServer *WsServer) handleSubcribeCrossCustomTokenPrivacyByPrivateKey(para
 		cResult <- RpcSubResult{Error: err}
 		return
 	}
-	privateKey := arrayParams[0].(string)
+	privateKey, ok := arrayParams[0].(string)
+	if !ok {
+		err := rpcservice.NewRPCError(rpcservice.RPCInvalidParamsError, errors.New("Params is invalid"))
+		cResult <- RpcSubResult{Error: err}
+		return
+	}
 	keyWallet, err := wallet.Base58CheckDeserialize(privateKey)
 	if err != nil {
 		err := rpcservice.NewRPCError(rpcservice.SubcribeError, err)

